webpack.config.js

```diff
@@ -11,23 +15,28 @@ module.exports = {
 
   output: {
     library: `[name]`,
-    filename: `[name].bundle.js`
+    filename: `[name].bundle.js`,
+    publicPath: commonPublicPath
+  },
+
+  resolve: {
+    alias: {
+      "react": path.resolve(`./node_modules/react`),
+      "react-dom": path.resolve(`./node_modules/react-dom`),
+      "styled-components": path.resolve(`./node_modules/styled-components`)
+    },
   },
 
   optimization: {
+    runtimeChunk: {
+       name: vendorsBundleName
+    },
     splitChunks: {
-      chunks: `all`,
-      minSize: 1,
       cacheGroups: {
-        facetedSearch: {
-          test: /[\\/]src[\\/]/,
-          name: `HomePagePanel`,
-          priority: -20
-        },
-        vendors: {
+        commons: {
           test: /[\\/]node_modules[\\/]/,
-          name: `vendors`,
-          priority: -10
+          name: vendorsBundleName,
+          chunks: 'all'
         }
       }
     }
```

Comment:
Examine the rules below and remove any which isn’t needed.

Reply:
This change is coming from the latest `atlas-package` repository.